src/EnumIface_TEST.cc

```diff
@@ -41,7 +41,7 @@ IGN_ENUM(myTypeIface, MyType, MY_TYPE_BEGIN, MY_TYPE_END,
 /////////////////////////////////////////////////
 TEST_F(EnumIfaceTest, StringCoversion)
 {
-  MyType type;
+  MyType type = MyType::TYPE2;
```

Comment:
Is this change related to the rest of the pull request?

Reply:
No, this is a separate commit fixing a compile warning.